_updates/2020-08-26-august.md

```diff
@@ -0,0 +1,53 @@
+---
+title: August 2020 / 2
+---
+
+## Bye bye summer, hello autumn!🍃🍂
+
+We have to say goodbye to summer, but something new has begun: autumn. The ["ember months"](https://en.wiktionary.org/wiki/ember_months) hold new challenges, suprises and of course new tasks for us. We face them with passion in the future as we have done so far. We are really excited of Hacktoberfest, we already started working on the contributor's tasks. In the upcoming months we will focus on bug fixin' instead of implementing new features.
```

Comment:
```suggestion
We have to say goodbye to summer, but it is also a new beginning of something: autumn. The ["ember months"](https://en.wiktionary.org/wiki/ember_months) hold new challenges, surprises and of course new tasks for us. We deal them with passion in the future as we have done so far. We are really excited about Hacktoberfest, so we already started to work on tasks for contributing. In the upcoming months we will focus on bug fixin' instead of implementing new features.
```